09-struct/conflation/main.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"fmt"
+)
+
+// START STRUCT OMIT
+type Circle struct {
+	X, Y, Radius int
+}
+// END STRUCT OMIT
+
+func main() { 
+// START BASIC OMIT
+	feet := 5.68
+	meters := 1.73
+	fmt.Println(feet + meters) // Is the result meaningful?
+
+	type Feet float64
+	type Meters float64
+	mile := Feet(5280)
+	km := Meters(1000)
+	// fmt.Println(mile + km) // type mismatch
```

Comment:
I thought it'd be useful to show the audience the compiler error, fix the problem, and then show the result, so I'm planning to edit all the compiler errors in my slides during the talk.

I'm inclined to uncomment the line but keep the comment noting the error.

I saw a message somewhere expressing concern that all the example code compiles, so I commented out anything that doesn't, but since the talks' code are all in different packages I'm not so sure it's important.